assets/js/index.js

```diff
@@ -0,0 +1,249 @@
+const API_KEY = "b248c5c1ee789732d38aa9fe65ef7935";
+const form = document.getElementById("form");
+const currentWeatherContainer = document.getElementById("currentWeather");
+const forecastWeatherContainer = document.getElementById("forecastWeather");
+let city = "";
+let lat = "";
+let lon = "";
+let temp = "";
+let icon = "";
+let wind = "";
+let humidity = "";
+let description = "";
+let tempDate = "";
+let uv = "";
+const renderRecentCities = () => {
+  const container = document.getElementById("recent");
+  const searchContainer = document.getElementById("list");
+  searchContainer.remove();
+  $("#recent").append(` <div id="list" class="list-group">
+            
+  </div>`);
+  // get recent cities from LS []
+  const recentCities = JSON.parse(localStorage.getItem("recentSearches"));
+  // if [] is empty then render alert
+  if (recentCities !== null) {
+    for (i = 0; i < recentCities.length; i++) {
+      let city = recentCities[i];
+      $("#list")
+        .append(`<a  data-value="${city}" id="${i}"class="list-group-item list-group-item-action">
+     ${city}
+    </a>
+  `);
+    }
+  }
+  // else render all recent cities
+  // add an event listener on div containing all cities
+  $("#list").click(handleRecentSubmit);
+};
+
+const renderCurrentWeather = (currentWeatherData) => {
+  // render the current weather data and append to section
+  let weatherContainer = document.getElementById("weatherContainer");
```

Comment:
More duplication here with declaring the variable on line 42 with vanilla javascript syntax, but then targeting it on line 43 with jquery.